Source/SPUD/Private/SpudSubsystem.cpp

```diff
@@ -219,13 +229,9 @@ void USpudSubsystem::OnPostLoadMap(UWorld* World)
 			SubscribeLevelObjectEvents(World->GetCurrentLevel());
 		}
 
-		// If we were loading, this is the completion
-		if (CurrentState == ESpudSystemState::LoadingGame)
-		{
-			LoadComplete(SlotNameInProgress, true);
-			UE_LOG(LogSpudSubsystem, Log, TEXT("Load: Success"));
-		}
-		
+		LoadComplete(SlotNameInProgress, true);
+		UE_LOG(LogSpudSubsystem, Log, TEXT("Load: Success"));
+
```

Comment:
As previous, this change is incorrect since it will break cases where a new map is loaded during normal play.